common.sh

```diff
@@ -262,6 +262,15 @@ export CI_TOKEN=${CI_TOKEN:-}
 set -x
 export PERSONAL_PULL_SECRET=${PERSONAL_PULL_SECRET:-$SCRIPTDIR/pull_secret.json}
 
+# Ensure working dir is always different than script dir. If not, some
+# files may get overriden during deployment process.
+if [ "$WORKING_DIR" == "$SCRIPTDIR" ]; then
```

Comment:
Could be useful to normalize the path:

```suggestion
if [ "$(realpath ${WORKING_DIR})" == "$(realpath ${SCRIPTDIR})" ]; then
```

Reply:
True, I also normalized it in the message for the user